guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -16,13 +16,19 @@ endif::[]
 * You must have root privileges on the host that you want to register.
 ifdef::satellite,orcharhino[]
 * You must have an activation key created.
+endif::[]
+ifdef::satellite[]
 * Optional: If you want to register hosts to Red{nbsp}Hat Insights, you must synchronize the `{RepoRHEL7Server}` repository and make it available in the activation key that you use.
 This is required to install the `insights-client` package on hosts.
+endif::[]
+ifdef::satellite,orcharhino[]
 * {ProjectServer}, any {SmartProxyServer}s, and all hosts must be synchronized with the same NTP server, and have a time synchronization tool enabled and running.
 * The daemon *rhsmcertd* must be running on the hosts.
 * An activation key must be available for the host.
 For more information, see {ContentManagementDocURL}Managing_Activation_Keys_content-management[Managing Activation Keys] in _{ContentManagementDocTitle}_.
 * Subscription Manager must be version 1.10 or later.
```

Comment:
I checked and CentOS 7 ships version 1.24.51. https://vault.centos.org/centos/6/contrib/x86_64/Packages/ shows that CentOS 6 contrib had version 1.20 so I wonder if listing this requirement even makes sense nowadays. I don't have a good overview of the various supported RHEL EUS versions and the subman version, but I think dropping it is fairly safe.

Reply:
I think only 6.10 is still under any kind of support, so yeah